### DISCUSSION TITLE
请问如何使用小于号<和大于号>翻页？

### DISCUSSION THREAD

**Discussion by vangoah20** [rime/home#874] [Q&A]
制作顶功方案，占用 了逗号和句号。拟使用<和>作为翻页键。
在key_binder添加 了：
    - { when: paging, accept: less, send: Paga_Up }
    - { when: has_menu, accept: greater, send: Page_Down }
 
无法实现翻页。
请问：是否需要其它配置项？还是说符号名称不对？

**Comment by lotem** [comment]
`Page_Up`拼錯了

**Comment by vangoah20** [comment]
感谢回复。
不好意思，笔误耽误您的时间。
不是page_up拼错的原因（一开始测试不通，以为是要用Prior和Next，也不行，换回来的时候拼错了）。
经测，如下配置都不能实现大于小于号翻页（仅列出小于号）：
- { when: paging, accept: less, send: Page_Up }
- { when: paging, accept: less, send: Prior }
- { when: has_menu, accept: less, send: Page_Up }
- { when: has_menu accept: less, send: Prior }



如下是所有binding配置。
    - { when: has_menu, accept: comma, send: comma }
    - { when: has_menu, accept: period, send: period }
    - { when: has_menu, accept: minus, send: minus }
    - { when: has_menu, accept: quote, send: quote }
    - { when: has_menu, accept: semicolon, send: semicolon }
    - { when: has_menu, accept: bracketleft, send: Prior }
    - { when: has_menu, accept: bracketright, send: Next }

    - { when: paging, accept: less, send: Page_Up }
    - { when: has_menu, accept: greater, send: Page_Down }

**Comment by lotem** [comment]
如果要按<kbd>Shift</kbd>鍵，給匹配的按鍵加上 `Shift+`
